portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,23 +31,55 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     public ArrayList<String> list = new ArrayList<String>();
+    ArrayList<String> comments = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    list.add("Abi");
-    list.add("Patchaiyappan");
-    list.add("19");
-    String json = "{";
-    json += "\"FirstName\": ";
-    json += "\"" + list.get(0) + "\"";
-    json += ", ";
-    json += "\"LastName\": ";
-    json += "\"" + list.get(1) + "\"";
-    json += ", ";
-    json += "\"Age\": ";
-    json += "\"" + list.get(2) + "\"";
-    json += "}";
-    response.setContentType("application/json");
-    response.getWriter().println(json);
+    //Query
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Task");
+
+    PreparedQuery results = datastore.prepare(query);
+
+    List<String> tasks = new ArrayList<>();
```

Comment:
Take a look at the example in the walkthrough: https://github.com/abiirami/software-product-sprint/tree/master/walkthroughs/week-2-server/examples/todo-list/src/main/java/com/google/sps Specifically at data/Task.java and servlets/ListTasksServlet.java. You'll need to create a new Comment.java and convert each entity to a Comment object and return a list of those Comments here.  Returning a list of strings technically works, but having this Comment object be an interface between the client + server is cleaner and builds a foundation for supporting more features like timestamped comments, comments with author names, etc.